src/Microsoft.AspNetCore.Dispatcher/Patterns/RoutePatternParser.cs

```diff
@@ -26,20 +26,16 @@ internal static class RoutePatternParser
             Asterisk
         };
 
-        public static RoutePattern Parse(string pattern)
+        public static RoutePattern Parse(string originalPattern)
```

Comment:
not digging the rename here

Reply:
Is it okay if I use the `trimmedPattern` after trimming the prefix and call this `pattern` again? I just need to make the distinction between the pattern that's the parameter and the one returned after trimming the leading '/' or "~/".

Reply:
Sure, that sounds good.